### PR TITLE
Feature.test l2adjacent without neutron 216

### LINKED ISSUE #216: There are no neutronless l2adjacent tests

### PR DESCRIPTION
**Pull request by zancas**
@mattgreene  
#### Implement standalone l2adjacent tests

    Issues:
    Fixes #216

    Problem: There were no standalone tests of the agent using l2adjacent service
    objects.

    Analysis:  They are added here the service objects were produced during a run
    of the l2adjacent tests.

    Tests: f5-openstack-agent/test/functional/neutronless/l2adjacent/test_create_loadbalancer.py

### CODE REVIEW

File: test/functional/neutronless/conftest.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+@pytest.fixture
+def setup_neutronless_test(request, bigip):
+    pretest_snapshot = frozenset(register_device(bigip))
+
+    def remove_test_created_elements():
+        for t in bigip.tm.net.fdb.tunnels.get_collection():
+            if t.name == 'tunnel-vxlan-45':
+                t.update(records=[])
```
Comment (mattgreene) [inline test/functional/neutronless/conftest.py]: See if you can leverage (or create) this as a method in the agent code.

File: test/functional/neutronless/l2adjacent/after__init__icd_opts.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    "allow_bulk": true, 
+    "allow_overlapping_ips": true, 
+    "allow_pagination": false, 
+    "allow_sorting": false, 
+    "api_extensions_path": "", 
+    "api_paste_config": "api-paste.ini", 
+    "auth_strategy": "keystone", 
+    "auth_version": "v3", 
+    "backlog": 4096, 
+    "barbican_endpoint": "http://10.190.4.169:9311",
```
Comment (mattgreene) [inline test/functional/neutronless/l2adjacent/after__init__icd_opts.json]: Replace IP with string, even though this setting is not used.

File: test/functional/neutronless/l2adjacent/test_loadbalancer_create.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
+         '~snat-traffic-group-local-only'
+         '-ce69e293-56e7-43b8-b51c-01b91d66af20_0?ver=11.6.0',
+
+         u'https://localhost/mgmt/tm/ltm/snatpool/'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.0',
+
+         u'https://localhost/mgmt/tm/net/fdb/tunnel/'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d~tunnel-vxlan-45?ver=11.5.0',
```
Comment (mattgreene) [inline test/functional/neutronless/l2adjacent/test_loadbalancer_create.py]: Research who is pinning the URI to a specific ver.  Is this a bug or expected?

File: test/functional/neutronless/l2adjacent/test_loadbalancer_create.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+         u'https://localhost/mgmt/tm/net/tunnels/tunnel/'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
+         '~tunnel-vxlan-45?ver=11.6.0',
+
+         u'https://localhost/mgmt/tm/sys/folder/'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.0'])
+
+
+class ConfFake(object):
```
Comment (mattgreene) [inline test/functional/neutronless/l2adjacent/test_loadbalancer_create.py]: See if this can be removed.

**Comment by zancas** [comment]
@mattgreene do you have any remaining concerns I need to address to merge this?

**Pull request merged**